libs/dh/shared/domain/src/lib/generated/v1/model/models.ts

```diff
@@ -14,7 +15,6 @@ export * from './create-actor-contact-dto';
 export * from './create-actor-dto';
 export * from './disconnection-type';
 export * from './eic-function';
-export * from './global-location-number-dto';
```

Comment:
nitpick: looks like the file itself is not used anymore. Can be deleted. This should be done manually because the auto-generator doesn't delete files.